askama/src/lib.rs

```diff
@@ -91,6 +91,9 @@ pub trait Template {
 
     /// Provides a conservative estimate of the expanded length of the rendered template
     const SIZE_HINT: usize;
+
+    /// The MIME type (Content-Type) of the data that gets rendered by this Template
+    const MIME_TYPE: &'static str;
```

Comment:
Should we deprecate `EXTENSION` and `extension()`? I feel like this is probably better.